xml/docu_styleguide.outline.xml

```diff
@@ -307,6 +296,24 @@
      </para>
     </listitem>
    </varlistentry>
+    <varlistentry>
+     <term>Contributors</term>
+     <listitem>
+      <para>
+       Writing the manuals is mostly a collaborative effort. To credit also
+       external contributors, add an appendix to the guides, based on the
+       <guimenu>Contributors</guimenu> page for the respective GitHub
+       repository. List the contributors in alphabetical order. Include a
+       link to the <guimenu>Contributors</guimenu> page on GitHub for the
+       most up-to-date version. For an example, see <xref
+        linkend="sec-contributors"/>.
+      </para>
+      <para>
+       For &suse; Best Practices and other one-off articles, credit the
```

Comment:
Not sure if this is about "one-off"s. It's more about articles (i.e. smaller documents) whose content is created and maintained by a single/very few individual outside the documentation team. If that ain't a snappy description, I don't know. :)

Reply:
Applied your proposal.